gems/aws-sdk-ec2/lib/aws-sdk-ec2.rb

```diff
@@ -14,6 +14,7 @@
 require_relative 'aws-sdk-ec2/client'
 require_relative 'aws-sdk-ec2/config'
 require_relative 'aws-sdk-ec2/errors'
+require_relative 'aws-sdk-ec2/endpoint'
```

Comment:
Wasn't this supposed to be endpoint_rules? Are you using the latest generation?

Reply:
the hearth module is `EndpointRules` but the name of the module in generated SDKs is just `Endpoints` (since I'd argue the rules aspect is an implementation detail there).